themes/bootstrap3/templates/search/advanced/layout.phtml

```diff
@@ -60,7 +60,8 @@
 ?>
 
 <?=$this->flashmessages()?>
-<form role="search" name="searchForm" id="advSearchForm" method="get" action="<?=$this->url($this->options->getSearchAction())?>">
+<div role="search">
+<form name="searchForm" id="advSearchForm" method="get" action="<?=$this->url($this->options->getSearchAction())?>">
```

Comment:
Ok, I thought it would be ok to just leave on the same level as the div because they will always be together, but I have no problem in changing that.
Thanks @demiankatz .